packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -573,9 +573,10 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   }
 
   void _updateSelectedContentIfNeeded() {
-    if (_lastSelectedContent?.plainText != _selectable?.getSelectedContent()?.plainText) {
+    if (_lastSelectedContent?.plainText !=_selectable?.getSelectedContent()?.plainText) {
       _lastSelectedContent = _selectable?.getSelectedContent();
-      widget.onSelectionChanged?.call(_lastSelectedContent);
+      final TextSelection? textSelection = _selectable?.textSelection;
```

Comment:
Will `_selectable?.getSelectedContent().textSelection` also have the global selection points? If so is there a reason why `SelectedContent` and `Selectable` both provide access to the text selection?

Reply:
SelectedContent is what is returned to the users; So in the case where we're building the selected content in a different way (like in the MultiSelectionDelegate) we're getting the aggregate of the text_selection of all the selectables. There is a meaningful distinction between the two.

Reply:
Following up on this comment.